TCA-467 - fix label for submit button

vas3a opened the pull request:
https://topcoder.atlassian.net/browse/TCA-467

Updates the label for the submit button for the project type challenges (eg. backend course).
Remove the "ctrl + enter" hint, as it is not listening to that. Instead it is just a simple form and by hitting enter (like in a normal form) will try to submit it. No additional hint was added as this is common sense (submitting forms by hitting enter).

vas3a committed:
TCA-467 - fix label for submit button

testflyjets commented:
@vas3a Does that fact that the code editor isn't listening for Ctrl-Enter mean it's broken? It normally does submit the code on those keystrokes (or Cmd-Enter on Mac). I'm wondering if we should alert the FCC guys to this. Your thoughts?

vas3a commented:
@testflyjets @brooketopcoder No, it's not borken. I added the label myself when I updated that form but I didn't realize it's not supposed to listen for ctrl+enter.

Ctrl+enter is used in the editor only -- because enter will just create a new line, so we need a "command" key to know we need to submit.

In a form, a simple enter submits the form.

vas3a commented:
To be clear. The issue in here is me adding the label that contains the "Ctrl+enter" . This is not supposed in any way to listen to ctrl+enter. 

Also, on FCC they don't have 2 states for the same button. They just have "submit", and when the tests are completed, you get the "completed modal".
We removed the completed modal so I had to add a 2nd state for the same button.
